README.md

```diff
@@ -65,22 +65,22 @@ expected to be similar to the daily releases.
 
 #### Build from sources
 
-See [Get Started Guide](/../sycl/sycl/doc/GetStartedGuide.md).
+See [Get Started Guide](./sycl/doc/GetStartedGuide.md).
 
 ### Report a problem
 
-Submit an [issue](/../../issues) or initiate a [discussion](/../../discussions).
+Submit an [issue](https://github.com/intel/llvm/issues) or initiate a [discussion](https://github.com/intel/llvm/discussions).
```

Comment:
I think the main goal of using relative links to make them work in forks automatically. These are working just fine in my fork.

Why do you use "absolute links" here?

Reply:
Normally, a fork will not clone issues and discussions. Which means that relative links wouldn't resolve in the fork.

Also note that an issue or discussion shouldn't be started in the fork anyway, it needs to be engaged on the original repository.

Reply:
> Normally, a fork will not clone issues and discussions. Which means that relative links wouldn't resolve in the fork.

These link to not point to any specific issues or discussions, so it doesn't matter if original repository issues/discussions are cloned to a fork.

Here is an example of a fork where it makes sense: https://github.com/arcs-skku/EMDC_llvm. There DPC++ evolves into some other direction and it looks like a separate project. It's better to report issues in that project to this project first.

Most of the forks are private developer forks and I don't think developers use these links in private repositories.

Reply:
I think we agree that those links won't work in private forks, given that the objects simply don't exist in forks with "default" settings:

![image](https://user-images.githubusercontent.com/54114043/228977256-4f149a73-2ffb-440d-a281-21d63fb24d4b.png)

So why not create links that just work, both in the original main repository, and any potential forks?

But I think you are considering the case where we have people who use the main repo as a starting point for an independent development project... And in that case, the links should point to that new project indeed.

I do believe that the usage of developers in their fork using the links to launch an issue or discussion for the main repository trumps the one-off situation where people want to start a new project. Those people just need to update those links, a one-time effort, and they are set for their project and corresponding forks.